app/assets/stylesheets/custom.scss

```diff
@@ -242,3 +242,17 @@
   width: 150px;
   height: 100px;
 }
+
+.checkbox {
+  margin-top: -10px;
+  margin-bottom: 10px;
+  span {
```

Comment:
Personally, I prefer to put an empty line before the nested component. For me, it is more readable.

Reply:
Good point, thanks!